lib/polymer/iron.dart

```diff
@@ -105,46 +49,40 @@ class CoreOverlay extends CoreElement {
   void close() => call('close');
 }
 
-abstract class CoreSelector extends CoreElement {
-  CoreSelector(String tag) : super(tag);
-  CoreSelector.from(HtmlElement element) : super.from(element);
+abstract class IronSelectableBehavior extends PolymerElement {
+  IronSelectableBehavior(String tag) : super(tag);
+  IronSelectableBehavior.from(HtmlElement element) : super.from(element);
 
   // TODO: add valueattr
 
   String get selected => "${property('selected')}";
-  set selected(String value) => setAttribute('selected', value);
+  set selected(String value) {
+    setAttribute('selected', value);
+  }
 
   dynamic get selectedIndex => property('selectedIndex');
 
   Object get selectedItem => property('selectedItem');
 
   /**
-   * Selects the previous item; this should be used in single selection only.
-   * This will wrap to the end if [wrapped] is `true` and it is already at the
-   * first item.
+   * Selects the previous item.
    */
-  void selectPrevious(bool wrapped) => call('selectPrevious', [wrapped]);
+  void selectPrevious() => call('selectPrevious');
 
   /**
-   * Selects the next item; this should be used in single selection only. This
-   * will wrap to the front if [wrapped] is `true` and it is already at the last
-   * item.
+   * Selects the next item.
    */
-  void selectNext(bool wrapped) => call('selectNext', [wrapped]);
+  void selectNext() => call('selectNext');
 
   /**
-   * Fired when an item's selection state is changed. This event is fired both
-   * when an item is selected or deselected.
+   *   The event that fires from items when they are selected.
+   *   Selectable will listen for this event from items and update
+   *   the selection state. Set to empty string to listen to no events.
    */
-  Stream get onCoreSelect => listen('core-select');
-
-  /**
-   * Fired when a selection is activated.
-   */
-  Stream get onCoreActivate => listen('core-activate');
+  Stream get ironActivate => listen('iron-activate');
 }
 
-class CoreSplitter extends CoreElement {
+class CoreSplitter extends PolymerElement {
```

Comment:
Is this still the latest? I can't find a splitter in https://elements.polymer-project.org/browse.